drafts/versions/02/draft-ietf-ippm-ioam-data-02.txt

```diff
@@ -1091,17 +1140,22 @@ Internet-Draft           In-situ OAM Data Fields           February 2018
 
 
 
-   IOAM POT Type:  7-bit identifier of a particular POT variant that
-      dictates the POT data that is included.  This document defines POT
-      Type 0:
+   IOAM POT Type:  8-bit identifier of a particular POT variant that
+      dictates the POT data that is included, to be set to 0 for the POT
```

Comment:
The wording is a little difficult to follow.

8-bit identifier of a particular POT variant that dictates the POT data that is included.  This section defines the POT data when the IOAM POT Type is set to the value 0.